packages/teams-js/test/public/video.spec.ts

```diff
@@ -3,18 +3,18 @@ import { app } from '../../src/public/app';
 import { FrameContexts } from '../../src/public/constants';
 import { video } from '../../src/public/video';
 import { Utils } from '../utils';
+import { DOMMessageEvent } from '../../src/internal/interfaces';
+
 /**
  * Test cases for selectPeople API
  */
 describe('video', () => {
-  const mobilePlatformMock = new FramelessPostMocks();
-  const utils = new Utils();
+  const framelessPlatformMock = new FramelessPostMocks();
+  const framedPlatformMock = new Utils();
 
   beforeEach(() => {
-    mobilePlatformMock.messages = [];
-
-    // Set a mock window for testing
-    app._initialize(mobilePlatformMock.mockWindow);
```

Comment:
It seems we still `app._uninitialize` in afterEach, did we want to keep the `app._initialize`? Either way, maybe we should have both of them or neither of them.

Reply:
We call `initializeWithContext` in every API, and in `initializeWithContext`  function, first thing we do is call app._initialize. In my opinion, this is how it should be becuase `app._initialize` call depends upon the client that we are mocking. For all the iframed clients, we want to use utils.mockWindow and for frameless, we want to use framelessPosttMock.mockWindow. So, I deleted it from top. 

we still need to uninitialize  to clear states between tests.

Reply:
Gotcha. That sounds good. :)